perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
+
+echo "Setting up the machine with Docker and Nvidia Driver"
+chmod +x ml_tests/tf/setup_scripts/setup_host.sh
+source ml_tests/tf/setup_scripts/setup_host.sh
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/"
+
+mkdir container_artifacts && mkdir container_artifacts/logs && mkdir container_artifacts/output
+
+# Building the dlc based dockerfile
+sudo docker build . -f perfmetrics/scripts/continuous_test/ml_tests/tf/Dockerfile -t dlc-gcsfuse --build-arg DLC_IMAGE_NAME=tf-gpu.2-10
```

Comment:
let's use tag (-t) as 'tf-dlc-gcsfuse' instead of 'dlc-gcsfuse' ? Also, change 'dlc-gcsufse:latest' to 'tf-dlc-gcsfuse:latest' in docker run command below.

Reply:
Done